modules/rpc/handler.go

```diff
@@ -130,6 +128,7 @@ func authorize(ctx context.Context, host service.Host) (fx.Context, error) {
 		host.Metrics().SubScope("rpc").SubScope("auth").Counter("fail").Inc(1)
 		fxctx.Logger().Error(auth.ErrAuthorization, "error", err)
 		// TODO(anup): GFM-255 update returned error to transport.BadRequestError (user error than server error)
+		// These ^ are internal YARPC errors, should we reference it?
```

Comment:
This is an open issue with YARPC team, with PR open - https://github.com/yarpc/yarpc-go/issues/687